sources/packages/backend/libs/test-utils/src/factories/application-exception-request.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  ApplicationException,
+  ApplicationExceptionRequest,
+  User,
+} from "@sims/sims-db";
+import * as faker from "faker";
+
+export function createFakeApplicationExceptionRequest(
+  applicationException: ApplicationException,
```

Comment:
I didn't get this idea of passing `applicationException` to create application exception.

Reply:
I didn't see any problem passing the whole object as we already have it. I prefer this way than having to do something like:
```applicationExceptionRequest.applicationException = { id: applicationExceptionId } ```
Any preferences @ann-aot @andrewsignori-aot @sh16011993 @guru-aot

Reply:
I was also following the same at first, then I got the review comment to use `{ id: dbObject.id} as dbObject`. Iam okay with both approaches, but in our project, we are following the `id` approach